_inc/client/at-a-glance/scan.jsx

```diff
@@ -184,54 +180,73 @@ class DashScan extends Component {
 		);
 	}
 
+	renderThreatsFound( numberOfThreats, dashboardUrl ) {
+		const { siteRawUrl } = this.props;
+		return (
+			<>
+				{ renderActiveCard( [
+					<h2 className="jp-dash-item__count is-alert">{ numberFormat( numberOfThreats ) }</h2>,
+					<p className="jp-dash-item__description">
+						{ __(
+							'Security threat found. Please {{fixThreats}}fix these{{/fixThreats}} as soon as possible.',
+							'Security threats found. Please {{fixThreats}}fix these{{/fixThreats}} as soon as possible.',
+							{
+								count: numberOfThreats,
+								components: {
+									fixThreats: <a href={ dashboardUrl } />,
```

Comment:
Should this open in a new tab since it's an external site?

Reply:
done